wgpu-core/src/command/compute.rs

```diff
@@ -42,9 +42,27 @@ pub enum ComputeCommand {
         buffer_id: id::BufferId,
         offset: BufferAddress,
     },
+    PushDebugGroup {
+        color: u32,
+        len: usize,
+        #[cfg_attr(any(feature = "trace", feature = "replay"), serde(skip))]
+        phantom_marker: PhantomSlice<u8>,
+    },
+    PopDebugGroup,
+    InsertDebugMarker {
+        color: u32,
+        len: usize,
+        #[cfg_attr(any(feature = "trace", feature = "replay"), serde(skip))]
+        phantom_marker: PhantomSlice<u8>,
+    },
     End,
 }
 
+#[derive(Debug)]
+struct State {
+    debug_scope_depth: u32,
```

Comment:
would you mind putting the `binder` in here, just like with `State` used in render passes? it would be good for consistency

Reply:
still looks unresolved